packages/headless/src/features/facets/category-facet-set/category-facet-set-slice.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from './category-facet-set-actions';
```

Comment:
I refactored following the initial PR comments. Here is a link to the [initial commit](https://github.com/coveo/ui-kit/pull/1214/commits/94e2fe178ed41a1a687a43d917e757f14cd1dba0) for a clearer diff of the behavioral changes in case it is helpful.